app/models/concerns/thredded/post_common.rb

```diff
@@ -54,9 +54,31 @@ def context_options
       {
         asset_root: Rails.application.config.action_controller.asset_host || '',
         post:       self,
+        whitelist:  sanitize_whitelist,
       }
     end
 
+    def sanitize_whitelist
+      HTML::Pipeline::SanitizationFilter::WHITELIST[:transformers] += [
+        lambda do |env|
+          node = env[:node]
+
+          a_tags = node.css('a')
```

Comment:
Instead of adding a transforming and allowing rel, let's use the `add_attributes` features:

```ruby
add_attributes: {
  'a' => {'target' => 'blank', 'rel' => 'nofollow noopener'}
}
```